db/schema.rb

```diff
@@ -129,6 +129,8 @@
     t.string "encrypted_file_data_iv"
     t.string "encrypted_evss_response"
     t.string "encrypted_evss_response_iv"
+    t.string "encrypted_bgs_responses"
```

Comment:
Knowing that this is for flashes specifically and that we will be querying this to know when the flashes job passes or fails and that we will soon be working on special issues in the same fashion, does it make sense to store seperate BGS responses for flashes and special issues? Being that it's an encrypted cell we won't really be able to query for when flashes vs special issues errors exist etc.

Reply:
i think it depends on what we will ultimately care about. i agree that if we're going to need to query, say, only flash errors over the period of years potentially, then yeah, separate fields is probably a must. if we plan more to check for errors over the period of days or weeks, its probably good enough to query all "bgs errors" and then split out by "type" for the report itself? also might make a difference whether we're primarily going to care about only flashes or only special issues at any given point, or if we're typically caring about "bgs errors" holistically.

happy to make this field more specific if that makes the most sense

Reply:
I went ahead with a more specific column name. if we're strongly considering needing a boolean for each flash and special_issues failures, then might as well keep the responses themselves separate.